pkgs/development/python-modules/json-rpc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonApplication
+, fetchPypi
+, tox
+}:
+buildPythonApplication rec {
+
+  name = "${pname}-${version}";
+  pname = "json-rpc";
+  version = "1.10.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06cg6gmznh5hf5p7078a8r63zmpzr5n25sq2rv5dhv3q49jxlymr";
+  };
+
+  buildInputs = [ tox ];
```

Comment:
You can use `checkInputs` instead.

Reply:
How will tox be used? Its preferred to run the actual test runner directly, instead of through `tox`.

Reply:
replaced it.